src/VecSim/algorithms/hnsw/graph_data_ram.cpp

```diff
@@ -0,0 +1,216 @@
+#include "graph_data_ram.h"
+struct LevelDataOnRam {
+    static size_t max_num_outgoing_links;
```

Comment:
Not sure that it can be static, since we may have several instances of HNSW index in redis with different `M` values